src/seedu/addressbook/commands/ThenListCommand.java

```diff
@@ -0,0 +1,38 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+public class ThenListCommand extends Command {
```

Comment:
Something to think about: Is `ThenListCommand` an actual command? Can the user explicitly execute it? This leads to the question: can you use inheritance here instead?
Can `ThenListCommand` be an abstract class or an interface?
`AddCommand` and `DeleteCommand` can be thought of as `ThenListCommand`s.